portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -26,39 +31,51 @@
 import java.util.List;
 import com.google.gson.Gson;
 
-/** Servlet that returns some example content. TODO: modify this file to handle comments data */
+/** Servlet that handles comments data */
 @WebServlet("/data")
 public class DataServlet extends HttpServlet {
   private static class Comment {
-    private String email, comment;
-    public Comment(String email, String comment){
+    private final String email, comment, time;
+    public Comment(String email, String comment, String time){
       this.email = email;
       this.comment = comment;
+      this.time = time;
     }
   }
-  private List<Comment> comments = new ArrayList<Comment>();
-
+  private Query comments_query = new Query("Comment").addSort("time", SortDirection.DESCENDING);
+  private DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
```

Comment:
I made this comment on another PR from someone else. I think this initialization might be nontrivial (for DatastoreService) and would suggest it be done as part of init.

```java
@Override
public void init() {
  datastore = DatastoreServiceFactory.getDatastoreService();
}
```

Reply:
could datastore be final?